### DISCUSSION TITLE
Is it possible to add cross-platform, linkable banners in AR mode?

### DISCUSSION THREAD

**Discussion by theranbrig** [google/model-viewer#1883] [Q&A]
Is it possible at this point to add external links while in VR mode for both Android and iOS?  I have tried using the method of appending parameters to the usdz file via 'https://example.com/file.usdz#applePayButtonType=plain' but nothing appears.  I have not been able to find anything that supports this thus far.

**Comment by cdata** [comment]
@theranbrig the closest you can probably get to this is to add a banner or something like that to the model itself, if you want it to be cross-platform. And, unfortunately, the banner will not work like a link.

If you create a .reality file using Apple's Reality Composer, I speculate that it would be possible to make links out of banners given this blurb in the docs: https://developer.apple.com/documentation/realitykit/creating_3d_content_with_reality_composer/building_custom_behaviors/adding_interactivity_to_behaviors#3379984

Of course, this is an advanced approach, and will not be cross-platform.

**Comment by cdata** [comment]
I should also add that once we land a stable WebXR-based AR mode, this will all become a lot easier.

Also, we recently landed a feature (#992) that will give you scripted access to the model's scene graph, which will make adding interaction very similar to how you do it on the web today.

All that to say, there is stuff coming own the pipe that will enable you to do what you are describing a lot more easily.
